target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -162,6 +162,7 @@ get_status_led() {
 		;;
 	hc5*61|\
 	hc5661a|\
+	hc5611|\
```

Comment:
This one is wrong. You don't have a `blue:system` LED. Please use the `green:power` LED instead.

Furthermore, please use `hiwifi,hc5611` as boardname everywhere.

Reply:
It's my mistake.